include/tvm/runtime/data_type.h

```diff
@@ -131,7 +150,7 @@ class DataType {
    */
   bool operator==(const DataType& other) const {
     return data_.code == other.data_.code && data_.bits == other.data_.bits &&
-           data_.lanes == other.data_.lanes;
+           data_.lanes == other.data_.lanes;  // && is_scalable_ == other.is_scalable_;
```

Comment:
seem like no diff with original code,  is the comment  for "is_scalable_ == other.is_scalable" intend logic?